.github/workflows/build-release.yml

```diff
@@ -7,11 +7,16 @@ jobs:
     uses: th2-net/.github/.github/workflows/compound-java.yml@main
     with:
       build-target: 'Sonatype'
+      integration-test-enabled: true
+      javaVersion: '11' # remove after update kotlin IllegalAccessError: superclass access check failed: class org.jetbrains.kotlin.kapt3.base.javac.KaptJavaCompiler (in unnamed module @0x3969f5fc) cannot access class com.sun.tools.javac.main.JavaCompiler (in module jdk.compiler) because module jdk.compiler does not export com.sun.tools.javac.main to unnamed module @0x3969f5fc
```

Comment:
note: this will be resolved once we update kotlin to `1.9.*` or above

Reply:
Also, we can add compile options to allow access to this model from kapt by adding `--add-exports ...` arguments to the compile tasks

Reply:
Do you have example with `--add-exports` ?

Reply:
Thank you